src/wallet/signer.ts

```diff
@@ -12,7 +12,7 @@ import { sign as signWithKeypair, verify } from 'ripple-keypairs'
 import { ValidationError } from '../common/errors'
 import { Signer } from '../models/common'
 import { Transaction } from '../models/transactions'
-import { verifyBaseTransaction } from '../models/transactions/common'
+import { verify as verifyTransaction } from '../models/transactions'
```

Comment:
Why not just leave it as `verify`?

Reply:
conflict w/ `ripple-keypairs.verify(....)`.

Reply:
Is it worth renaming the function itself?